translator/trace/jaeger/protospan_to_jaegerproto.go

```diff
@@ -412,6 +412,12 @@ func appendJaegerTagFromOCStatusProto(jTags []jaeger.KeyValue, ocStatus *tracepb
 		VType:  jaeger.ValueType_INT64,
 	})
 
+	jTags = append(jTags, jaeger.KeyValue{
```

Comment:
I suggest that we only append TagError if ocStatus.Code != 0. If I remember "error=false" is not added to the tags since absence of error is equivalent to "error=false".

Reply:
Done.